next/pages/vzn/[slug].tsx

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+
+import { ParsedUrlQuery } from 'node:querystring'
+
+import { GeneralQuery, RegulationEntityFragment } from '@backend/graphql'
+import { client } from '@backend/graphql/gql'
+import PageLayout from '@components/layouts/PageLayout'
+import RegulationDetailPageContent from '@components/pages/RegulationDetailPageContent'
+import { LocalizationsProvider } from '@components/providers/LocalizationsProvider'
+import PageHeader from '@components/ui/PageHeader/PageHeader'
+import { SectionContainer } from '@components/ui/SectionContainer/SectionContainer'
+import { GeneralContextProvider } from '@utils/generalContext'
+import { GetStaticPaths, GetStaticProps } from 'next'
+import Head from 'next/head'
+import * as React from 'react'
+
+interface StaticParams extends ParsedUrlQuery {
+  slug: string
+}
+
+type RegulationPageProps = {
+  general: GeneralQuery
+  regulation: RegulationEntityFragment
+}
+
+export const getStaticPaths: GetStaticPaths<StaticParams> = async () => {
+  const { regulations } = await client.allRegulations()
+
+  const paths = (regulations?.data ?? [])
+    .filter((regulation) => regulation?.attributes?.slug)
+    .map((regulation) => ({
+      params: {
+        slug: regulation?.attributes?.slug ?? '',
```

Comment:
Here you can use `regulation.attributes.slug!` and disable eslint rule for this line, because you filtered only regulations that have slug (on line 33).